### PR TITLE
Refactor SCAN projects into config file

### PR DESCRIPTION
**Pull request by truong97**
The SCAN projects are refactored into SFS classic REDCap projects. 

Some design implementation to keep in mind:

  - The SCAN barcodes rules are still maintained from `parse_scan_redcap` where mail-in kits only use `return_utm_barcode` and kiosk/in-person uses, in order `utm_tube_barcode_2`,  `reenter_barcode`, and `reenter_barcode_2`. Was unsure if the censoring manually-entered barcode needs to be refactored or not. 

- With the elimination of the standalone SCAN export, the classic export (with the refactored SCAN) will act as the base "redcap_data" dataframe and the longitudinal export will be amended to it.

Also note that SCAN positives/inconclusives are automatically sent to the portal (aka no`pending`) as intended.

### CODE REVIEW

**Comment by truong97** [comment]
In terms of testing:

- tested on prod and local
- ran `generate-results-csv` and dumped the results into a json file on local computer for both branch and master
- compared the two files via `diff` (note that the ordering is different for the two files due to the argument ordering for `transform`.
- no differences between the two (aside from the ordering noted above)

Note that I did not have access to two longitudinal projects (yakima en and yakima es) when performing testing, thus were excluded from the output files.

**Comment by truong97** [comment]
[Refactor: SCAN into RoR Config File](https://trello.com/c/Ue0xCWq0/1305-refactor-scan-into-ror-config-file)

**Review by kschwabefry — APPROVED** [review]
This is a huge improvement to have this refactored. It reads so much cleaner and more consistent. Left one question below about mail in kits and censoring participant-entered barcodes.
Also confirming, I tested locally on master and feature branches and got the same results as well (minus ordering changes).

File: bin/return-of-results/transform [hunk 1]
```diff
@@ -24,84 +24,6 @@ logging.captureWarnings(True)
...
-    # reenter_barcode.  We *do not* use participant-entered barcodes for mailed
-    # kits, so those get pre-emptively blanked.
-    #
-    # For our kiosk/in-person collections, utm_tube_barcode_2 is scanned by
-    # staff.  If scanning doesn't work, then a manually-entered barcode is in
-    # reenter_barcode; it is double-entered into reenter_barcode_2.
-    collection_via = redcap_data["project_collection_via"]
-
-    redcap_data.loc[collection_via != "kiosk", "utm_tube_barcode_2"] = pd.NA
-    redcap_data.loc[collection_via != "kiosk", "reenter_barcode"]    = pd.NA
```
[the remark this answers is not in the document]
Comment (truong97) [reply bin/return-of-results/transform:72]: Yes correct! The config file only uses `return_utm_barcode` for the mail-in kits (there is only `return_utm_barcode` in the prioritized barcode list)

**Comment by truong97** [comment]
Deployed.

**Pull request merged**